lms/djangoapps/instructor/enrollment.py

```diff
@@ -287,6 +294,41 @@ def _reset_module_attempts(studentmodule):
     studentmodule.save()
 
 
+def _fire_score_changed_for_block(course_id, student, block, module_state_key):
+    """
+    Fires a SCORE_CHANGED event for the given module. The earned points are
+    always zero. We must retrieve the possible points from the XModule, as
+    noted below.
+    """
+    cache = FieldDataCache.cache_for_descriptor_descendents(
+        course_id=course_id,
+        user=student,
+        descriptor=block,
+        depth=0
+    )
+    # For implementation reasons, we need to pull the max_score from the XModule, even though the data is not user-
+    # specific.  Here we bind the data to the current user.
+    request = RequestFactory().get('/dummy-collect-max-grade')
```

Comment:
We shouldn't have production code creating mock requests.
Rather, you can have the test code create a mock request before calling this method.